sycl/include/sycl/ext/oneapi/matrix/matrix-aot-amx.hpp

```diff
@@ -38,7 +38,7 @@
 __SYCL_INLINE_NAMESPACE(cl) {
 namespace sycl {
 namespace ext {
-namespace intel {
+namespace oneapi {
```

Comment:
BTW, @bader @dkhaldi , should we move this file to ext/intel directory since it works on intel HW?

Reply:
mistakenly change it, sorry for inconvience.

Reply:
> BTW, @bader @dkhaldi , should we move this file to ext/intel directory since it works on intel HW?

Please keep everything in ext/oneapi.
The feature is still oneAPI "matrix". It is just that the initial implementation targeted only AMX.